packages/hoprd/src/api/v2.ts

```diff
@@ -77,20 +85,27 @@ export async function setupRestApi(
       peerId: (input) => {
         try {
           // this call will throw if the input is no peer id
-          return !!PeerId.createFromB58String(input)
-        } catch (_err) {
```

Comment:
`_err` indicates that the variable is not used, so we can also do `catch {` since `err` is not used

Reply:
The problem is the `!!expression` construct. Returning `true` or `false` is way more clear syntactically and less error-prone semantically. Especially when looking at different Typescript transpilation targets